### PR TITLE
Improve staking UX: ask for OLAS once

### PR DESCRIPTION
**Pull request by jmoreira-valory**
Improve staking UX: ask for OLAS once.

### CODE REVIEW

File: run_service.sh [hunk 1]
```diff
@@ -300,7 +300,7 @@ create_storage() {
     # Generate the RPC file
     echo -n "$rpc" > "../$rpc_path"
 
-    # Generate the operator's key
+    # Generate the owner/operator's key
```
Comment (DavidMinarsch) [inline run_service.sh:303]: Why are we adding "owner" everywhere now?

Comment (jmoreira-valory) [reply run_service.sh:303]: Because we use a single address to act as the service owner and agent operator.

Comment (DavidMinarsch) [reply run_service.sh:303]: Ok

**Review by dagacha — APPROVED** [review]
LGTM

**Pull request merged**